setup.py

```diff
@@ -14,6 +14,7 @@
     install_requires=[
         'acme',
         'cryptography',
+        "ipaddress; python_version < '3.3'",
```

Comment:
This is a backport of the Python 3.3 `ipaddress` module. It is actually already required by `uritools` and `cryptography`.